src/content/blog/using-frankenphp-with-ddev.md

```diff
@@ -0,0 +1,134 @@
+---
+title: "Using FrankenPHP with DDEV"
+pubDate: 2025-07-02
+#modifiedDate: 2025-07-03
+summary: Maintaining an add-on involves regularly updating it to stay compatible with new features in both the upstream ddev-addon-template and DDEV itself.
+author: Stas Zhuk
+featureImage:
+  src: /img/blog/2025/07/frankenphp-logo.png
+  alt: FrankenPHP logo
+  credit: FrankenPHP logo from [GitHub](https://github.com/php/frankenphp)
+categories:
+  - Guides
+---
+
+## Introduction
+
+The PHP ecosystem is changing fast, with tools like FrankenPHP [FrankenPHP](https://frankenphp.dev) improving both performance and developer experience.
+
+This guide explains two ways to integrate FrankenPHP, based on my experience.
+
+You can either run FrankenPHP as a separate service (lets you install extra PHP extensions) or inside the `web` container (uses a static binary without support for extra extensions).
+
+## DDEV FrankenPHP Add-on
+
+I created the [stasadev/ddev-frankenphp](https://github.com/stasadev/ddev-frankenphp) add-on to run FrankenPHP as a separate service with:
+
+- Ability to install PHP extensions (Redis, Xdebug, SPX, etc.)
+- Better resource isolation
+
+### ⚙️ Installation:
+
+```bash
+ddev config --webserver-type=generic
+ddev add-on get stasadev/ddev-frankenphp
+ddev restart
+```
+
+### ⚠️ Limitations:
+
+- Tools from the `web` container are not available
+- `ddev xdebug` and `ddev launch` don't work (target `web` container)
+- Enabling/disabling Xdebug requires container rebuild
+
+If you want to suggest some feature or found a bug, feel free to [open an issue](https://github.com/stasadev/ddev-frankenphp/issues).
+
+## Running FrankenPHP in the Web Container
+
+Alternatively, FrankenPHP can be run inside the `web` container. This quickstart shows an example (for a Drupal 11 project) where FrankenPHP is added as an extra daemon.
```

Comment:
Just add a note and a link saying "from the DDEV quickstart"

Reply:
> And note that it's just one example of providing your own webserver instead of using the built-in webserver.

I added a small section about this https://20250701-stasadev-frankenphp.ddev-com-front-end.pages.dev/blog/using-frankenphp-with-ddev/#generic-webserver